### ISSUE TITLE
About company-childframe

### ISSUE THREAD

**Issue by tumashu** [company-mode/company-mode#753]
In my opinion, company-childframe has been stable enough for common use, 
do you think merge it to company or  build it as a independent melpa package?

**Comment by dgutov** [comment]
Please publish it separately for now, and collect the first round of user bug reports.

I've seen a couple of glitches (more minor ones than we've discussed previously), but haven't had the time to turn them into proper reports, since they are not easy to reproduce.

**Comment by tumashu** [comment]
ok

**Comment by stardiviner** [comment]
Hi, where is the `company-childframe` source code? I have not found it anywhere.

**Comment by mambolevis** [comment]
Let me ask you one question:

I'm using emacs version: `GNU Emacs 25.3.1 (x86_64-w64-mingw32)` and when I use `M-x package-list-packages`  company-childframe appears incompatible.
![image](https://user-images.githubusercontent.com/13024208/37242720-904b5e44-246e-11e8-881a-6dd68e6c55f7.png)
Do you know why this message appears even though I'm not trying to installed `company-childframe` package?

By the way, I had the same situation with org, it appeared incompatible. I found a solution just by adding the following in my init file:

`(add-to-list 'package-archives '("org" . "https://orgmode.org/elpa/") t)`

**Comment by tumashu** [comment]
posframe  need emacs 26

**Comment by tumashu** [comment]
company-childframe need posframe

**Comment by mambolevis** [comment]
> posframe need emacs 26
> company-childframe need posframe

Yes, I read that `posframe` needs emacs-26. The point why is it shown as incompatible when I am not trying to use it? I don't have neither `posframe` nor `company-childframe` in my init file.

**Comment by dgutov** [comment]
It means you can't install it.

**Comment by mambolevis** [comment]
Ahh, ok. I see. I think that org creates a bit confusion regarding incompatible packages.  Org is also shown as incompatible, but I solve the problem adding the following `(add-to-list 'package-archives '("org" . "https://orgmode.org/elpa/") t)`

**Comment by dgutov** [comment]
After you do that, it's not listed as "incompatible", is it? I don't see much of a confusion.

**Comment by miskatul-anwar** [comment]
for Doom emacs:
```lisp
 lsp:  
  (company +childframe) 
```

**Comment by dgutov** [comment]
Now included: https://github.com/company-mode/company-mode/pull/1525 Thanks all.
